src/rp2_common/hardware_flash/include/hardware/flash.h

```diff
@@ -56,6 +56,24 @@
 extern "C" {
 #endif
 
+/*! \brief  Initialise QSPI interface and external QSPI devices for execute-in-place
+ *  \ingroup hardware_flash
+ *
+ * This function performs the same first-time flash setup that would normally occur during startup
```

Comment:
maybe we should be a bit clearer about what "during startup" means and how it differs between RP2040 and RP2350